CI/tests/my_tests

```diff
@@ -1,12 +1 @@
-test_pods
-test_nodes
-test_time
-test_app_outages
-test_container
-test_zone
-test_io_hog
-test_mem_hog
-test_cpu_hog
-test_shut_down
 test_net_chaos
```

Comment:
Are we running the network chaos test ( https://github.com/redhat-chaos/krkn/blob/main/CI/scenarios/network_chaos.yaml ) on Kind?

Reply:
Yes, it passes there.